main/src/com/google/refine/model/recon/ReconciledDataExtensionJob.java

```diff
@@ -250,11 +251,13 @@ protected ReconciledDataExtensionJob.DataExtension collectResult(
                     int v = val.get("int").asInt();
                     storeCell(rows, rowindex, colindex, v);
                 } else if (val.has("date")) {
-                    ToDate td = new ToDate();
-                    String[] args = new String[1];
-                    args[0] = val.get("date").asText();
-                    Object v = td.call(null, args);
-                    storeCell(rows, rowindex, colindex, v);
+                    Object date;
+                    try {
+                        date = CalendarParser.parseAsOffsetDateTime(val.get("date").asText());
```

Comment:
This also seems unrelated, but even if it gets moved to another PR, I don't think we should be introducing direct calls to CalendarParser which we should be targeting to get rid of. Who's date format is this? If it's something like an ISO date, we should just use the Java classes directly. If it's a human date, it should probably stay wrapped in `ToDate()`

Reply:
The motivation is still the same: removing a dependency from a core class to a GREL function. Using `CalendarParser` means that we support almost the same sort of date formats as before, which could be useful for services which rely on that.
But that being said now that the specs mandate ISO dates (although only in the JSON schemas so far, we should fix that) it should probably be safe to switch to supporting ISO dates only.

Reply:
Sorry for the delay in circling back on this. I think using Java classes to support ISO dates (only) would be a better approach.